NEWS.md

```diff
@@ -1,3 +1,8 @@
+# mcstate 0.5.9
+
+* Add `nested_step_ratio` parameter to `pmcmc_control` for controlling the ratio of fixed:varied
```

Comment:
.md files are one exception to 80 char rule - we get most consistent formatting by soft-wrapping